flexget/plugins/input/my_anime_list.py

```diff
@@ -90,7 +91,8 @@ def on_task_input(self, task, config):
                         url="https://myanimelist.net" + anime["anime_url"],
                         mal_name=anime["anime_title"],
                         mal_poster=anime["anime_image_path"],
-                        mal_type=anime["anime_media_type_string"]
+                        mal_type=anime["anime_media_type_string"],
+                        tags=anime["tags"]
```

Comment:
This should probably be prefixed with `mal_`

Reply:
Good point, I'll change that.